tests/pages/firefox_dnt.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class FirefoxDNTPage(BasePage):
+
+    _url = '{base_url}/{locale}/firefox/dnt'
+
+    _dnt_status = (By.CSS_SELECTOR, '#dnt-status > h5')
+    _faq_1_header = (By.ID, 'faq-1-header')
+    _faq_1_panel = (By.ID, 'faq-1-panel')
+
+    def dnt_status_is_visible(self):
+        el = self.selenium.find_element(*self._dnt_status)
+        return el.is_displayed()
+
+    def click_faq_1_header(self):
+        self.selenium.find_element(*self._faq_1_header).click()
+        Wait(self.selenium, self.timeout).until(
+            expected.visibility_of_element_located(self._faq_1_panel))
+
+    def faq_1_panel_is_visible(self):
+        el = self.selenium.find_element(*self._faq_1_panel)
+        return el.is_displayed()
```

Comment:
I think using this pattern sounds certainly more flexible, good suggestions :+1:  I think it is perhaps overkill to repeat clicking on every item, as it is essentially testing the same code multiple times, but I think opening / closing a section should be a minimum requirement for a base test.